context/user/UserState.js

```diff
@@ -171,12 +179,125 @@ const EmailState = (props) => {
 		});
 	};
 
-	const setUserNotFound = () => dispatch({ type: "USER_NOT_FOUND" });
+	const createUserGasless = async (address, web3Provider) => {
```

Comment:
Remove redundant codes from gasless functions and create separate functions for them